public/countdown.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css"
+    />
+    <title>Countdown to Summer of Making</title>
+    <style>
+      body {
+        margin: 0px;
+        font-family: Phantom Sans;
```

Comment:
Recommend doing `Phantom Sans, system-ui, sans-serif` so the text displays even if the font fails to load
```suggestion
        font-family: Phantom Sans, system-ui, sans-serif;
```